data.json

```diff
@@ -1158,10 +1158,10 @@
     "issueNumber": 602
   },
   {
-    "name": "Noorin Sakhi",
-    "githubUsername": "NoorinSakhi",
-    "imageUrl": "https://user-images.githubusercontent.com/70807500/126067761-2fd771c0-0c5c-4748-ab0d-429250027830.jpg",
-    "issueNumber": 604
+    "name": "CodingSpecies",
+    "githubUsername": "CodingSpecies",
+    "imageUrl": "https://user-images.githubusercontent.com/70807500/129594715-ef05ecc0-720f-4b75-a507-c2afed68cf02.png",
```

Comment:
I suggest more of an image like this.

![image](https://user-images.githubusercontent.com/51391473/129598330-ca71253a-4431-4bbd-91af-8db083144b9a.png)

```suggestion
    "imageUrl": "https://user-images.githubusercontent.com/51391473/129598330-ca71253a-4431-4bbd-91af-8db083144b9a.png",
```